README.md

```diff
@@ -31,6 +31,29 @@ The library provides a set of functions to interact with the testcontainers. A s
 
 (tc/stop! container)
 ```
+
+To create a container using your local Dockerfile 
```

Comment:
Great, that you enhanced documentation, too! But you added the function call to the example part, the more important one is the section below, with a detailed explanation of input parameters and return values, could you add it there, too?

Reply:
Could you give some example as how you like it be done?

Reply:
Yes of course. You can see examples in line 59 of this file for the `create` function, or in line 93 for the `start` function. It is usually the function name, a short description, the input-parameters and output data structure as a table each and a short example on the usage of the function